src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx

```diff
@@ -127,6 +184,28 @@ const PatientView = (props: {
                             triggerAsc: 'Sorter i stigende rekkefølge',
                             triggerDesc: 'Sorter i synkende rekkefølge',
                         }}
+                        rowSelection={
+                            compareSchemesMode
+                                ? {
+                                      type: 'checkbox',
+                                      hideSelectAll: true,
+                                      onChange: (selectedRowKeys: ReactText[]) => {
+                                          if (selectedRowKeys.length <= MAX_VALUE) {
+                                              setComparingSchemesIds(selectedRowKeys);
+                                              setReachedMaxValue(false);
+                                          } else if (selectedRowKeys.length > MAX_VALUE) setReachedMaxValue(true);
```

Comment:
I suggest changing `else if` to `else` as they are equivalent here, and this change will simplify the code.

Reply:
Yes, forgot to go back to just else while I was looking for the warning bug